docs/guide/cleanup.md

```diff
@@ -1,10 +1,10 @@
-# Cleanup
+# Clean up
```

Comment:
Please revert this and changes to speedup/levelup

Reply:
See above